src/main/scala/uk/gov/hmrc/play/audit/filters/AuditFilter.scala

```diff
@@ -37,79 +39,75 @@ trait AuditFilter extends EssentialFilter with HttpAuditEvent {
 
   def controllerNeedsAuditing(controllerName: String): Boolean
 
+  implicit def mat: Materializer
+
   val maxBodySize = 32665
 
   protected def needsAuditing(request: RequestHeader): Boolean =
-    (for (controllerName <- request.tags.get(Routes.ROUTE_CONTROLLER))
+    (for (controllerName <- request.tags.get(play.routing.Router.Tags.ROUTE_CONTROLLER))
       yield controllerNeedsAuditing(controllerName)).getOrElse(true)
 
   protected def getBody(result: Result) = {
-    val bytesToString: Enumeratee[Array[Byte], String] = Enumeratee.map[Array[Byte]] { bytes => new String(bytes) }
-    val consume: Iteratee[String, String] = Iteratee.consume[String]()
-    result.body |>>> bytesToString &>> consume
-  }
-
-  protected def captureRequestBody(next: Iteratee[Array[Byte], Result], onDone: Promise[Array[Byte]]): Iteratee[Array[Byte], Result] = {
-    def step(body: mutable.ArrayBuffer[Byte], nextI: Iteratee[Array[Byte], Result])(input: Input[Array[Byte]]): Iteratee[Array[Byte], Result] = {
-      input match {
-        case Input.El(e) => {
-          val newBody = if (body.length > maxBodySize) {
-            Logger.warn(s"txm play auditing: sanity check ${body.length} exceeds maxLength ${maxBodySize} - do you need to be auditing this payload?")
-            body
-          } else body ++= e.take(maxBodySize - body.length)
-          Cont[Array[Byte], Result](step(newBody, Iteratee.flatten(nextI.feed(Input.El(e)))))
-        }
-        case Input.Empty => Cont[Array[Byte], Result](step(body, nextI))
-        case Input.EOF => {
-          val result = Iteratee.flatten(nextI.feed(Input.EOF))
-          onDone.success(body.toArray)
-          result
+    val sink = Sink.fold[String, ByteString]("")({
```

Comment:
The BodyCaptor is for capturing the request body (and truncating as required)m getBody gets the response body - do you want me to name them a little clearer..?